### PR TITLE
Daily PR June 18

### PR DESCRIPTION
**Pull request by pe7r**
Work with tests for app.

### CODE REVIEW

File: src/Components/Pagination/Pagination.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        expect(shallowToJson(output)).toMatchSnapshot()
+    })
+
+    it('Pagination goes on next page', () => {
+        const spy = jest.fn();
+        const output = mount(
+            <Pagination actualPage={1} totalCount={879} changingPage={spy}/>
+        )
+        output.find('[name="buttonNext"]').simulate('click');
+        expect(spy.mock.calls.length).toBe(1)
```
Comment (ruslanpalagin) [inline src/Components/Pagination/Pagination.test.js]: Please add also argument checking

expect(spy.mock.calls[0][0]).toBe('next')

File: src/Components/SearchInput/SearchInput.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import { shallow } from 'enzyme';
+import { shallowToJson } from 'enzyme-to-json';
+
+
+
+describe('SearchInput', () => {
+    describe('props works correctly', () => {
+        it('theme is an empty string', () => {
+            const output = shallow(
+                <SearchInput placeholder="Search" theme=''/>
```
Comment (ruslanpalagin) [inline src/Components/SearchInput/SearchInput.test.js]: Try to be consistent using " and '
placeholder="Search" theme=""
OR
placeholder='Search' theme=''

File: src/tempPolyfills.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const raf = (global.requestAnimationFrame = cb => {
```
Comment (ruslanpalagin) [inline src/tempPolyfills.js]: What is it?)

Comment (pe7r) [reply src/tempPolyfills.js]: I get it from youtube lesson. I was trying everything he showed to connect jest and enzyme.

Comment (pe7r) [reply src/tempPolyfills.js]: Do I have to delete this?

File: src/Components/SearchInput/SearchInput.test.js (hunk 2 above)
Comment (pe7r) [reply src/Components/SearchInput/SearchInput.test.js]: Fixed.

File: src/Components/Pagination/Pagination.test.js (hunk 1 above)
Comment (pe7r) [reply src/Components/Pagination/Pagination.test.js]: Added for both tests prev/next

File: src/tempPolyfills.js (hunk 3 above)
Comment (ruslanpalagin) [reply src/tempPolyfills.js]: Yes, such thing are confusing

**Pull request merged**